boards/riscv/tlsr9518adk80d/tlsr9518adk80d.dts

```diff
@@ -119,7 +119,7 @@
 		};
 		storage_partition: partition@f0000 {
 			label = "storage";
-			reg = <0x000f0000 0x00010000>;
+			reg = <0x000f0000 0x00008000>;
```

Comment:
Would you mind explaining the need for this change in the commit message?

Reply:
Added explanation comment directly to the dts file
 ```
/* region 0x000f8000 .. 0x000FFFFF
  * is reserved for factory calibration
  */
```

Reply:
This seems like the sort of thing that belongs in its own commit then. It's a change to the partition layout not related to the commit's purpose, which is to "add Telink B91 Flash driver support". But it's not a big deal; not blocking over this.